MYOB.API.SDK/SDK.Package/Package.nuspec

```diff
@@ -23,19 +23,19 @@
     <copyright>Copyright MYOB Technology Pty. Ltd. 2013-2015</copyright>
     <dependencies>
       <group targetFramework="net35">
-        <dependency id="Newtonsoft.Json" version="[7.0.1]" />
+        <dependency id="Newtonsoft.Json" version="7.0.1" />
```

Comment:
@NikhitaA, was this file part of the change? It looks like it's a sync issue between perforce and github, and for this file, I would say that github's version is the most up to date

Reply:
@jaspermondie This file was not manually updated as part of our changes. So, I will revert this file to reflect the github change.